cross-files/aarch64-darwin

```diff
@@ -0,0 +1,27 @@
+
+[host_machine]
+system = 'darwin'
+cpu_family = 'arm'
+cpu = 'aarch64'
+endian = 'little'
+
+[properties]
+
+
+[binaries]
+c = 'clang'
+cpp = 'clang++'
+objc = 'clang'
+objcpp = 'clang++'
+ar = 'ar'
+ld = 'ld'
+
+[built-in options]
+c_args = ['-arch', 'arm64']
+cpp_args = ['-arch', 'arm64']
+objc_args = ['-arch', 'arm64']
+objcpp_args = ['-arch', 'arm64']
+c_link_args = ['-arch', 'arm64']
+cpp_link_args = ['-arch', 'arm64']
+objc_link__args = ['-arch', 'arm64']
```

Comment:
Seems like the double underscore might be a typo.

Reply:
Indeed, so this is not needed - I'll leave it in for future reference.